app/src/components/main/analytics/events/ga4Events/profilePageEvent.js

```diff
@@ -14,18 +14,57 @@
  * limitations under the License.
  */
 
+import { getBasicClickEventParameters } from '../common/ga4Utils';
+
 const PROFILE_PAGE = 'profile';
 
-const BASIC_EVENT_PARAMETERS_PROFILE = {
-  action: 'click',
-  category: PROFILE_PAGE,
-  place: PROFILE_PAGE,
-};
+const BASIC_EVENT_PARAMETERS_PROFILE = getBasicClickEventParameters(PROFILE_PAGE);
 
-export const PROFILE_EVENT = {
+export const PROFILE_EVENTS = {
   CHANGE_LANGUAGE: (lang) => ({
     ...BASIC_EVENT_PARAMETERS_PROFILE,
+    place: PROFILE_PAGE,
     type: lang,
     element_name: 'language_drop_down',
   }),
+
+  CLICK_API_KEYS_TAB_EVENT: {
+    ...BASIC_EVENT_PARAMETERS_PROFILE,
+    element_name: 'tab_api_keys',
+  },
+
+  CLICK_GENERATE_API_KEY_BUTTON: (isApiKey) => ({
```

Comment:
Let's use camelCase for event creators.

Reply:
CONSTANT_CASE is used everywhere in event creators.

Reply:
When migrating to GA4, we agreed to use CamelCase for event creators to better distinguish constant events from them.
So if there are some events with old naming that affected by particular task, let's provide new names for them according to above agreement.